test/integration_test.py

```diff
@@ -396,6 +398,21 @@ def __check_manifest(self, file: IO[bytes], uuid_field_name: str) -> List[Mappin
         self.assertEqual(bundle_uuid, str(uuid.UUID(bundle_uuid)))
         return rows
 
+    def _check_curl_manifest(self, _catalog: CatalogName, response: bytes):
+        text = TextIOWrapper(BytesIO(response))
+        num_files = 0
+        lines = (
+            line.rstrip()
+            for line in text.readlines()
+            if not line == '\n' and not line.startswith('--')
```

Comment:
```suggestion
            if line != '\n' and not line.startswith('--')
```

It seems a bit odd that you rstrip each line (indicating that leading space is insignificant) but that the conditional would be thrown off by it. I'd remove the rstrip. Might also want to add a comment explaining what the intent is of the conditional. I assume `--` are comments?